docs/source/app_developers_guide/creating_sawtooth_network.rst

```diff
@@ -289,16 +363,60 @@ Step 1: Configure the Network on the First Node
 
         --peers tcp://172.0.1.3,tcp://172.0.1.4
```

Comment:
Agreed.  I assume that this is the "public endpoint string", aka the ``--endpoint`` option, for the OTHER validator node(s). So, for this current example, I think it would be something like this:

     --peers tcp://172.0.1.3:8800,tcp://172.0.1.4:8800

Question: Would it be more helpful to show pretend host names instead of example IP addresses?  Such as: 

--peers tcp://MyPeer2:8800,tcp://MyPeer3:8800